portal/home.php

```diff
@@ -24,8 +24,6 @@
 
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Twig\TwigContainer;
-use OpenEMR\Events\PatientPortal\RenderEvent;
-use OpenEMR\Events\PatientPortal\AppointmentFilterEvent;
```

Comment:
looks like this may be reverting recent code (also several more blocks were reverted below in this script)

Reply:
Thanks nice catch